python/ray/dataframe/dataframe.py

```diff
@@ -430,10 +430,26 @@ def between_time(self, start_time, end_time, include_start=True,
         raise NotImplementedError("Not Yet implemented.")
 
     def bfill(self, axis=None, inplace=False, limit=None, downcast=None):
-        raise NotImplementedError("Not Yet implemented.")
+        return self._map_partitions(lambda df: df.bfill(axis=axis,
```

Comment:
I think that this will have the same issues ffill has, where it won't backfill to other partitions. Also, I think limit might be tricky to implement as well. If we set limit to some number, it will put the limit on each partition, but not the ray df as a whole.

Reply:
Yep. That's why I "de-implemented" bfill in https://github.com/ray-project/ray/pull/1495/commits/153c0b523a7c9945bb004935e999da68dd79e56c

Can's we just use the scheme we have discussed in previous meeting?

Reply:
Oh, I see. sorry, I didn't see that. Yes, I implement fillna and ffill with what we discussed in the last meeting, which should take care of method. I'm still trying to figure out how to deal with limit. Right now, I'm thinking of counting all NaNs in each column and specifying a range of partitions to run on for each column. I'm not sure if that's the best approach though.